inference-engine/tests_deprecated/functional/vpu/common/layers/myriad_layers_exp_generateproposals_test.hpp

```diff
@@ -140,7 +140,11 @@ static void genInputs(InferenceEngine::BlobMap inputMap,
     inputIMinfo[1] = PrecisionUtils::f32tof16( (float) imgW );
 }
 
+#ifdef __APPLE__
+TEST_P(myriadLayersTestsExpGenerateProposals_smoke, DISABLED_ExpGenerateProposals) {
```

Comment:
Why did you disable some tests for macOS? Is it related with RTTI issues?

Reply:
This tests fails on MacOS, but this problem is depended on input data, which generates randomly, so it very hard to reproduce, and since those are deprecated tests, they were disabled. As far as I can tell they don't relate to the RTTI issues

Reply:
Maybe some variable or class member isn't initialised and it causes the issue... Do you have a plan to move this test on new infrastructure and enable it?

Reply:
there is plans for moving tests, but with a very low priority